### PR TITLE
[WXP] (Ribbon APIs) Clarify host and platform support for Ribbon APIs

### PR DESCRIPTION
**Pull request by AlexJerabek**
The application and platform support for the Ribbon APIs as incorrectly listed. This PR fixes that (along with the related conceptual PR).

### CODE REVIEW

File: docs/requirement-sets/common/ribbon-api-requirement-sets.md [hunk 1]
```diff
@@ -12,31 +12,16 @@ Requirement sets are named groups of API members. Office Add-ins use requirement
...
-
-> [!IMPORTANT]
->
-> - The RibbonApi requirement sets are supported only on task pane add-ins.
-> - The RibbonApi requirement sets are supported for production add-ins only in Excel.
-> - RibbonApi 1.1 (not 1.2) is available as a preview in PowerPoint and Word, but only in Office on Windows (Microsoft 365 subscription) and Office on Mac. It's also available for preview in PowerPoint on the web. To understand how to access Office.js preview APIs, see [Accessing the Office JavaScript API library](/office/dev/add-ins/develop/understanding-the-javascript-api-for-office#accessing-the-office-javascript-api-library).
-
-## Support for version 1.1 in Office on Windows (Microsoft 365 subscription)
+| RibbonApi 1.2 | Suported | Version 2102 (Build 13801.20294) | Version 2102 (Build 13801.20294) | Office 2021: Version 2108 (Build 14326.20454) | 16.53.806.0 | Not supported |
+| RibbonApi 1.1 | In preview* | 12527.20880 | Version 2006 (Build 13001.20266) | Office 2021: Version 2108 (Build 14326.20454) | 16.38 | Not supported |
```
Comment (samantharamon) [inline docs/requirement-sets/common/ribbon-api-requirement-sets.md:22]:
```suggestion
| RibbonApi 1.1 | In preview* | Version 2002 (12527.20880) | Version 2006 (Build 13001.20266) | Office 2021: Version 2108 (Build 14326.20454) | 16.38 | Not supported |
```

File: docs/requirement-sets/common/ribbon-api-requirement-sets.md [hunk 2]
```diff
@@ -12,31 +12,16 @@ Requirement sets are named groups of API members. Office Add-ins use requirement
...
-| RibbonApi 1.1 | Supported | See [support](#support-for-version-11-in-office-on-windows-microsoft-365-subscription)<br>[section](#support-for-version-11-in-office-on-windows-microsoft-365-subscription) | Version 2006 (Build 13001.20266) | Office 2021: Version 2108 (Build 14326.20454) | 16.38 | Not supported |
-
-> [!IMPORTANT]
->
-> - The RibbonApi requirement sets are supported only on task pane add-ins.
-> - The RibbonApi requirement sets are supported for production add-ins only in Excel.
-> - RibbonApi 1.1 (not 1.2) is available as a preview in PowerPoint and Word, but only in Office on Windows (Microsoft 365 subscription) and Office on Mac. It's also available for preview in PowerPoint on the web. To understand how to access Office.js preview APIs, see [Accessing the Office JavaScript API library](/office/dev/add-ins/develop/understanding-the-javascript-api-for-office#accessing-the-office-javascript-api-library).
-
-## Support for version 1.1 in Office on Windows (Microsoft 365 subscription)
+| RibbonApi 1.2 | Suported | Version 2102 (Build 13801.20294) | Version 2102 (Build 13801.20294) | Office 2021: Version 2108 (Build 14326.20454) | 16.53.806.0 | Not supported |
```
Comment (samantharamon) [inline docs/requirement-sets/common/ribbon-api-requirement-sets.md:21]:
```suggestion
| RibbonApi 1.2 | Supported | Version 2102 (Build 13801.20294) | Version 2102 (Build 13801.20294) | Office 2021: Version 2108 (Build 14326.20454) | 16.53.806.0 | Not supported |
```

File: docs/requirement-sets/requirement-sets.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ To work as expected, your Office Add-in might depend on a specific Office applic
...
...requirement-sets.md)</li><li>[ImageCoercion 1.1](common/image-coercion-requirement-sets.md#imagecoercion-11)</li><li>[ImageCoercion 1.2](common/image-coercion-requirement-sets.md#imagecoercion-12)</li><li>[KeyboardShortcuts 1.1](common/keyboard-shortcuts-requirement-sets.md)</li><li>[MatrixBindings](common/office-add-in-requirement-sets.md#matrixbindings)</li><li>[MatrixCoercion](common/office-add-in-requirement-sets.md#matrixcoercion)</li><li>[OpenBrowserWindowApi 1.1](common/open-browser-window-api-requirement-sets.md)</li><li>[RibbonApi 1.1](common/ribbon-api-requirement-sets.md)</li><li>[Selection](common/office-add-in-requirement-sets.md#selection)</li><li>[Settings](common/office-add-in-requirement-sets.md#settings)</li><li>[SharedRuntime 1.1](common/shared-runtime-requirement-sets.md)</li><li>[SharedRuntime 1.2](common/shared-runtime-requirement-sets.md)</li><li>[TableBindings](common/office-add-in-requirement-sets.md#tablebindings)</li><li>[TableCoercion](common/office-add-in-requirement-sets.md#tablecoercion)</li><li>[TextBindings](common/office-add-in-requirement-sets.md#textbindings)</li><li>[TextCoercion](common/office-add-in-requirement-sets.md#textcoercion)</li></ul>|
```
Comment (samantharamon) [inline docs/requirement-sets/requirement-sets.md:24]:
```suggestion
|Office on Windows</li><li>Microsoft 365 subscription</li><li>retail perpetual Office 2016 and later</li></ul>|<ul><li>TaskPane</li><li>Content</li><li>[CustomFunctions](excel/custom-functions-requirement-sets.md)</li><li>[Add-in Commands](common/add-in-commands-requirement-sets.md)</li></ul>|<ul><li>[ExcelApi 1.1](excel/excel-api-1-1-requirement-set.md)</li><li>[ExcelApi 1.2](excel/excel-api-1-2-requirement-set.md)</li><li>[ExcelApi 1.3](excel/excel-api-1-3-requirement-set.md)</li><li>[ExcelApi 1.4](excel/excel-api-1-4-requirement-set.md)</li><li>[ExcelApi 1.5](excel/excel-api-1-5-requirement-set.md)</li><li>[ExcelApi 1.6](excel/excel-api-1-6-requirement-set.md)</li><li>[ExcelApi 1.7](excel/excel-api-1-7-requirement-set.md)</li><li>[ExcelApi 1.8](excel/excel-api-1-8-requirement-set.md)</li><li>[ExcelApi 1.9](excel/excel-api-1-9-requirement-set.md)</li><li>[ExcelApi 1.10](excel/excel-api-1-10-requirement-set.md)</li><li>[ExcelApi 1.11](excel/excel-api-1-11-requirement-set.md)</li><li>[ExcelApi 1.12](excel/excel-api-1-12-requirement-set.md)</li><li>[ExcelApi 1.13](excel/excel-api-1-13-requirement-set.md)</li><li>[ExcelApi 1.14](excel/excel-api-1-14-requirement-set.md)</li><li>[ExcelApi 1.15](excel/excel-api-1-15-requirement-set.md)</li><li>[ExcelApi 1.16](excel/excel-api-1-16-requirement-set.md)</li><li>[ExcelApi 1.17](excel/excel-api-1-17-requirement-set.md)</li></ul>|<ul><li>[BindingEvents](common/office-add-in-requirement-sets.md#bindingevents)</li><li>[CompressedFile](common/office-add-in-requirement-sets.md#compressedfile)</li><li>[DialogApi 1.1](common/dialog-api-requirement-sets.md)</li><li>[DialogApi 1.2](common/dialog-api-requirement-sets.md)</li><li>[DocumentEvents](common/office-add-in-requirement-sets.md#documentevents)</li><li>[File](common/office-add-in-requirement-sets.md#file)</li><li>[IdentityAPI 1.3](common/identity-api-requirement-sets.md)</li><li>[ImageCoercion 1.1](common/image-coercion-requirement-sets.md#imagecoercion-11)</li><li>[ImageCoercion 1.2](common/image-coercion-requirement-sets.md#imagecoercion-12)</li><li>[KeyboardShortcuts 1.1](common/keyboard-shortcuts-requirement-sets.md)</li><li>[MatrixBindings](common/office-add-in-requirement-sets.md#matrixbindings)</li><li>[MatrixCoercion](common/office-add-in-requirement-sets.md#matrixcoercion)</li><li>[OpenBrowserWindowApi 1.1](common/open-browser-window-api-requirement-sets.md)</li><li>[RibbonApi 1.1](common/ribbon-api-requirement-sets.md)</li><li>[RibbonApi 1.2](common/ribbon-api-requirement-sets.md)</li><li>[Selection](common/office-add-in-requirement-sets.md#selection)</li><li>[Settings](common/office-add-in-requirement-sets.md#settings)</li><li>[SharedRuntime 1.1](common/shared-runtime-requirement-sets.md)</li><li>[SharedRuntime 1.2](common/shared-runtime-requirement-sets.md)</li><li>[TableBindings](common/office-add-in-requirement-sets.md#tablebindings)</li><li>[TableCoercion](common/office-add-in-requirement-sets.md#tablecoercion)</li><li>[TextBindings](common/office-add-in-requirement-sets.md#textbindings)</li><li>[TextCoercion](common/office-add-in-requirement-sets.md#textcoercion)</li></ul>|
```
Added RibbonApi 1.2. This should also be added to the Office on Mac row.

**Review by samantharamon — CHANGES_REQUESTED** [review]

**Comment by AlexJerabek** [comment]
Thanks for catching all that @samantharamon!

File: docs/requirement-sets/requirement-sets.md [hunk 4]
```diff
@@ -20,8 +20,8 @@ To work as expected, your Office Add-in might depend on a specific Office applic
...
...-sets.md)</li><li>[DialogApi 1.1](common/dialog-api-requirement-sets.md)</li><li>[DialogApi 1.2](common/dialog-api-requirement-sets.md)</li><li>[DocumentEvents](common/office-add-in-requirement-sets.md#documentevents)</li><li>[File](common/office-add-in-requirement-sets.md#file)</li><li>[IdentityAPI 1.3](common/identity-api-requirement-sets.md)</li><li>[KeyboardShortcuts 1.1](common/keyboard-shortcuts-requirement-sets.md)</li><li>[MatrixBindings](common/office-add-in-requirement-sets.md#matrixbindings)</li><li>[MatrixCoercion](common/office-add-in-requirement-sets.md#matrixcoercion)</li><li>[Selection](common/office-add-in-requirement-sets.md#selection)</li><li>[Settings](common/office-add-in-requirement-sets.md#settings)</li><li>[SharedRuntime 1.1](common/shared-runtime-requirement-sets.md)</li><li>[SharedRuntime 1.2](common/shared-runtime-requirement-sets.md)</li><li>[TableBindings](common/office-add-in-requirement-sets.md#tablebindings)</li><li>[TableCoercion](common/office-add-in-requirement-sets.md#tablecoercion)</li><li>[TextBindings](common/office-add-in-requirement-sets.md#textbindings)</li><li>[TextCoercion](common/office-add-in-requirement-sets.md#textcoercion)</li></ul>|
```
Comment (samantharamon) [inline docs/requirement-sets/requirement-sets.md:23]:
```suggestion
|Office on the web|<ul><li>TaskPane</li><li>Content</li><li>[CustomFunctions](excel/custom-functions-requirement-sets.md)</li><li>[Add-in Commands](common/add-in-commands-requirement-sets.md)</li></ul>|<ul><li>[ExcelApi 1.1](excel/excel-api-1-1-requirement-set.md)</li><li>[ExcelApi 1.2](excel/excel-api-1-2-requirement-set.md)</li><li>[ExcelApi 1.3](excel/excel-api-1-3-requirement-set.md)</li><li>[ExcelApi 1.4](excel/excel-api-1-4-requirement-set.md)</li><li>[ExcelApi 1.5](excel/excel-api-1-5-requirement-set.md)</li><li>[ExcelApi 1.6](excel/excel-api-1-6-requirement-set.md)</li><li>[ExcelApi 1.7](excel/excel-api-1-7-requirement-set.md)</li><li>[ExcelApi 1.8](excel/excel-api-1-8-requirement-set.md)</li><li>[ExcelApi 1.9](excel/excel-api-1-9-requirement-set.md)</li><li>[ExcelApi 1.10](excel/excel-api-1-10-requirement-set.md)</li><li>[ExcelApi 1.11](excel/excel-api-1-11-requirement-set.md)</li><li>[ExcelApi 1.12](excel/excel-api-1-12-requirement-set.md)</li><li>[ExcelApi 1.13](excel/excel-api-1-13-requirement-set.md)</li><li>[ExcelApi 1.14](excel/excel-api-1-14-requirement-set.md)</li><li>[ExcelApi 1.15](excel/excel-api-1-15-requirement-set.md)</li><li>[ExcelApi 1.16](excel/excel-api-1-16-requirement-set.md)</li><li>[ExcelApi 1.17](excel/excel-api-1-17-requirement-set.md)</li><li>[ExcelApiOnline](excel/excel-api-online-requirement-set.md)</li></ul>|<ul><li>[BindingEvents](common/office-add-in-requirement-sets.md#bindingevents)</li><li>[CompressedFile](common/office-add-in-requirement-sets.md#compressedfile)</li><li>[DevicePermissionService 1.1](common/device-permission-service-requirement-sets.md)</li><li>[DialogApi 1.1](common/dialog-api-requirement-sets.md)</li><li>[DialogApi 1.2](common/dialog-api-requirement-sets.md)</li><li>[DocumentEvents](common/office-add-in-requirement-sets.md#documentevents)</li><li>[File](common/office-add-in-requirement-sets.md#file)</li><li>[IdentityAPI 1.3](common/identity-api-requirement-sets.md)</li><li>[KeyboardShortcuts 1.1](common/keyboard-shortcuts-requirement-sets.md)</li><li>[MatrixBindings](common/office-add-in-requirement-sets.md#matrixbindings)</li><li>[MatrixCoercion](common/office-add-in-requirement-sets.md#matrixcoercion)</li><li>[RibbonApi 1.2](common/ribbon-api-requirement-sets.md)</li><li>[Selection](common/office-add-in-requirement-sets.md#selection)</li><li>[Settings](common/office-add-in-requirement-sets.md#settings)</li><li>[SharedRuntime 1.1](common/shared-runtime-requirement-sets.md)</li><li>[SharedRuntime 1.2](common/shared-runtime-requirement-sets.md)</li><li>[TableBindings](common/office-add-in-requirement-sets.md#tablebindings)</li><li>[TableCoercion](common/office-add-in-requirement-sets.md#tablecoercion)</li><li>[TextBindings](common/office-add-in-requirement-sets.md#textbindings)</li><li>[TextCoercion](common/office-add-in-requirement-sets.md#textcoercion)</li></ul>|
```

File: docs/requirement-sets/requirement-sets.md [hunk 5]
```diff
@@ -20,8 +20,8 @@ To work as expected, your Office Add-in might depend on a specific Office applic
...
...dd-in-requirement-sets.md#documentevents)</li><li>[File](common/office-add-in-requirement-sets.md#file)</li><li>[IdentityAPI 1.3](common/identity-api-requirement-sets.md)</li><li>[ImageCoercion 1.1](common/image-coercion-requirement-sets.md#imagecoercion-11)</li><li>[ImageCoercion 1.2](common/image-coercion-requirement-sets.md#imagecoercion-12)</li><li>[MatrixBindings](common/office-add-in-requirement-sets.md#matrixbindings)</li><li>[MatrixCoercion](common/office-add-in-requirement-sets.md#matrixcoercion)</li><li>[OpenBrowserWindowApi 1.1](common/open-browser-window-api-requirement-sets.md)</li><li>[RibbonApi 1.1](common/ribbon-api-requirement-sets.md)</li><li>[Selection](common/office-add-in-requirement-sets.md#selection)</li><li>[Settings](common/office-add-in-requirement-sets.md#settings)</li><li>[SharedRuntime 1.1](common/shared-runtime-requirement-sets.md)</li><li>[TableBindings](common/office-add-in-requirement-sets.md#tablebindings)</li><li>[TableCoercion](common/office-add-in-requirement-sets.md#tablecoercion)</li><li>[TextBindings](common/office-add-in-requirement-sets.md#textbindings)</li><li>[TextCoercion](common/office-add-in-requirement-sets.md#textcoercion)</li></ul>|
```
Comment (samantharamon) [inline docs/requirement-sets/requirement-sets.md:25]:
```suggestion
|Office 2021 on Windows<br>(volume-licensed perpetual)|<ul><li>TaskPane</li><li>Content</li><li>[CustomFunctions](excel/custom-functions-requirement-sets.md)</li><li>[Add-in Commands](common/add-in-commands-requirement-sets.md)</li></ul>|<ul><li>[ExcelApi 1.1](excel/excel-api-1-1-requirement-set.md)</li><li>[ExcelApi 1.2](excel/excel-api-1-2-requirement-set.md)</li><li>[ExcelApi 1.3](excel/excel-api-1-3-requirement-set.md)</li><li>[ExcelApi 1.4](excel/excel-api-1-4-requirement-set.md)</li><li>[ExcelApi 1.5](excel/excel-api-1-5-requirement-set.md)</li><li>[ExcelApi 1.6](excel/excel-api-1-6-requirement-set.md)</li><li>[ExcelApi 1.7](excel/excel-api-1-7-requirement-set.md)</li><li>[ExcelApi 1.8](excel/excel-api-1-8-requirement-set.md)</li><li>[ExcelApi 1.9](excel/excel-api-1-9-requirement-set.md)</li><li>[ExcelApi 1.10](excel/excel-api-1-10-requirement-set.md)</li><li>[ExcelApi 1.11](excel/excel-api-1-11-requirement-set.md)</li><li>[ExcelApi 1.12](excel/excel-api-1-12-requirement-set.md)</li><li>[ExcelApi 1.13](excel/excel-api-1-13-requirement-set.md)</li><li>[ExcelApi 1.14](excel/excel-api-1-14-requirement-set.md)</li></ul>|<ul><li>[BindingEvents](common/office-add-in-requirement-sets.md#bindingevents)</li><li>[CompressedFile](common/office-add-in-requirement-sets.md#compressedfile)</li><li>[DialogApi 1.1](common/dialog-api-requirement-sets.md)</li><li>[DialogApi 1.2](common/dialog-api-requirement-sets.md)</li><li>[DocumentEvents](common/office-add-in-requirement-sets.md#documentevents)</li><li>[File](common/office-add-in-requirement-sets.md#file)</li><li>[IdentityAPI 1.3](common/identity-api-requirement-sets.md)</li><li>[ImageCoercion 1.1](common/image-coercion-requirement-sets.md#imagecoercion-11)</li><li>[ImageCoercion 1.2](common/image-coercion-requirement-sets.md#imagecoercion-12)</li><li>[MatrixBindings](common/office-add-in-requirement-sets.md#matrixbindings)</li><li>[MatrixCoercion](common/office-add-in-requirement-sets.md#matrixcoercion)</li><li>[OpenBrowserWindowApi 1.1](common/open-browser-window-api-requirement-sets.md)</li><li>[RibbonApi 1.1](common/ribbon-api-requirement-sets.md)</li><li>[RibbonApi 1.2](common/ribbon-api-requirement-sets.md)</li><li>[Selection](common/office-add-in-requirement-sets.md#selection)</li><li>[Settings](common/office-add-in-requirement-sets.md#settings)</li><li>[SharedRuntime 1.1](common/shared-runtime-requirement-sets.md)</li><li>[TableBindings](common/office-add-in-requirement-sets.md#tablebindings)</li><li>[TableCoercion](common/office-add-in-requirement-sets.md#tablecoercion)</li><li>[TextBindings](common/office-add-in-requirement-sets.md#textbindings)</li><li>[TextCoercion](common/office-add-in-requirement-sets.md#textcoercion)</li></ul>|
```

File: docs/requirement-sets/requirement-sets.md [hunk 6]
```diff
@@ -20,8 +20,8 @@ To work as expected, your Office Add-in might depend on a specific Office applic
...
...>TaskPane</li><li>Content</li></ul>|<ul><li>[ExcelApi 1.1](excel/excel-api-1-1-requirement-set.md)</li></ul>|<ul><li>[BindingEvents](common/office-add-in-requirement-sets.md#bindingevents)</li><li>[CompressedFile](common/office-add-in-requirement-sets.md#compressedfile)</li><li>[DialogApi 1.1](common/dialog-api-requirement-sets.md)\*</li><li>[DocumentEvents](common/office-add-in-requirement-sets.md#documentevents)</li><li>[File](common/office-add-in-requirement-sets.md#file)</li><li>[ImageCoercion 1.1](common/image-coercion-requirement-sets.md#imagecoercion-11)</li><li>[MatrixBindings](common/office-add-in-requirement-sets.md#matrixbindings)</li><li>[MatrixCoercion](common/office-add-in-requirement-sets.md#matrixcoercion)</li><li>[Selection](common/office-add-in-requirement-sets.md#selection)</li><li>[Settings](common/office-add-in-requirement-sets.md#settings)</li><li>[TableBindings](common/office-add-in-requirement-sets.md#tablebindings)</li><li>[TableCoercion](common/office-add-in-requirement-sets.md#tablecoercion)</li><li>[TextBindings](common/office-add-in-requirement-sets.md#textbindings)</li><li>[TextCoercion](common/office-add-in-requirement-sets.md#textcoercion)</li></ul>|
```
Comment (samantharamon) [inline docs/requirement-sets/requirement-sets.md:27]: I'm unable to add a comment to the Office on Mac row of the Excel table. We should also add RibbonApi 1.2 there.

**Review by samantharamon — APPROVED** [review]
I included a few other spots we need to add RibbonAPI 1.2 support to, but otherwise, changes look good. Thanks for making these changes.

**Pull request merged**